apps/opik-backend/config.yml

```diff
@@ -65,3 +65,6 @@ server:
   enableVirtualThreads: ${ENABLE_VIRTUAL_THREADS:-false}
   gzip:
     enabled: true
+
+openTelemetry:
+  disabled: ${OTEL_SDK_DISABLED:-true}
```

Comment:
I suggest doing the reverse condition, that means `enabled` instead of `disabled`. That's usually the pattern in our configuration file and also in Dropwizard. It's also easier to think about for the human brain.

Then, you can and probably start with `enabled: false` by default. We can change to be enabled by default in the future.

Minor: please add an extra empty new line at the end of this file.

Reply:
Same:

https://github.com/comet-ml/opik/pull/254#discussion_r1763974728